src/test/java/seedu/address/logic/parser/RemoveTagCommandParserTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.Test;
+
+import seedu.address.logic.commands.RemoveTagCommand;
+import seedu.address.model.tag.Tag;
+
+/**
+ * As we are only doing white-box testing, our test cases do not cover path variations
+ * outside of the RemoveTagCommand code. For example, inputs "friend" and "friend 1" take the
+ * same path through the RemoveTagCommand, and therefore we test only one of them.
+ * The path variation for those two cases occur inside the ParserUtil, and
+ * therefore should be covered by the ParserUtilTest.
+ */
+public class RemoveTagCommandParserTest {
```

Comment:
Should have more test cases here, this is where you can performs some regex testing too.
Does tag allow whitespaces, integers or symbols?

Update: Just saw the disclaimer on white-box testing, I'll just leave this comment here.